packages/ui/src/components/MessageInput/index.native.tsx

```diff
@@ -349,7 +355,6 @@ export const MessageInput = forwardRef<MessageInputHandle, MessageInputProps>(
 
               // @ts-expect-error setContent does accept JSONContent
               editor.setContent(newJson);
-              // editor.setSelection(initialSelection.from, initialSelection.to);
```

Comment:
Intentional?

Reply:
Was old/commented out code that we didn't need. Was leftover from something I had tried a while ago.